scrapy/commands/parse.py

```diff
@@ -51,7 +51,7 @@ def add_options(self, parser):
         parser.add_argument("--cbkwargs", dest="cbkwargs",
                             help="inject extra callback kwargs into the Request, it must be a valid raw json string")
         parser.add_argument("-d", "--depth", dest="depth", type=int, default=1,
-                            help="maximum depth for parsing requests [default: %default]")
+                            help="maximum depth for parsing requests [default: 1]")
```

Comment:
```suggestion
                            help="maximum depth for parsing requests [default: %(default)s]")
```
This way you don't need to define twice. Syntax is from documentation<sup>[1](https://docs.python.org/3/library/argparse.html)</sup>. Tested locally and it works!

> Replace strings with implicit arguments such as %default or %prog with the standard Python syntax to use dictionaries to format strings, that is, %(default)s and %(prog)s.

Reply:
I fixed thanks for the suggestion.